### ISSUE TITLE
FreeBSD apps plugin missing group

### ISSUE THREAD

**Issue by thiagoftsm** [netdata/netdata#7087]
##### Bug report summary
During the weekend I executed different test using FreeBSD and I notice that apps.plugin is not working correctly on it.
During the majority of the tests I ran a benchmark using wrk on Netdata and I got the followiing output when Netdata starts:

![Netdata_starts](https://user-images.githubusercontent.com/49162938/66768411-0ce5a500-eea2-11e9-849a-309cc464a86c.png)

But, after few minutes I have this output:

![Netdata_after](https://user-images.githubusercontent.com/49162938/66768421-140cb300-eea2-11e9-9ca1-cc62fae9ad04.png)

But, `htop` was showing me Netdata using cpu:

![Netdata_after3](https://user-images.githubusercontent.com/49162938/66768470-2dadfa80-eea2-11e9-9d75-ca6f36a47ba4.png)

To confirm that the problem was not only in my computer, I ran the following command in another computer:

```sh
 curl -o data2 https://mechani.se/netdata/api/v1/data?chart=apps.cpu
```

And the output was Netdata 100% with zeros.

##### OS / Environment
FreeBSD 12.0-RELEASE-p10
##### Netdata version (ouput of `netdata -V`)
netdata v1.18.0-21-g909fce2e
##### Component Name
apps.plugin
##### Steps To Reproduce
Start Netdata with all plugins enabled and let it running and run `wrk` on it during more than 10 minutes.
##### Expected behavior
To have charts showing Netdata on apps.cpu chart.

**Comment by thiagoftsm** [comment]
Another important detail, when I run the following debug command:

```sh
./apps.plugin debug no-users no-groups no-files 
```

the chart improves, but it is not 100% correct:

![Netdata_after10](https://user-images.githubusercontent.com/49162938/66768751-da887780-eea2-11e9-8f7e-658023ab9ed4.png)

**Comment by amoss** [comment]
There are some internal inconsistencies in the cpu metrics, these screenshots are from mechani.se . As you can see the time-slice measurements do not match the cpu-time. I will leave the agent running on https://mechani.se/netdata/ in-case more data is required. It was installed today from the nightly build with `sudo ./netdata-installer --install /usr/local`.
![cpuuse1](https://user-images.githubusercontent.com/1043609/66771109-a79dae00-eeb9-11e9-9b05-1b16be1cd65b.png)
![cpuuse2](https://user-images.githubusercontent.com/1043609/66771119-ad938f00-eeb9-11e9-9661-c9fdd767ca17.png)

**Comment by mfundul** [comment]
What `memory mode` did you test @thiagoftsm ? Does the same problem exist for both `save` and `dbengine`?

**Comment by thiagoftsm** [comment]
I used dbengine for all the tests, my main goal was to test whether the fixes released last week were enough, and during the tests I found this.
I will check the results for save and I will update here in few minutes.

**Comment by thiagoftsm** [comment]
I ran now with `save` mode, firstly I only started Netdata without to run the benchmark on it(`wrk`) and I already saw that I got better results:

![Netdata_after13](https://user-images.githubusercontent.com/49162938/66783567-ad01f500-eec7-11e9-9619-8519e314fcc4.png)

After this I decided to run the wrk to check whether `netdata` would appear with a better frequency and I also got better results:

![Netdata_after14](https://user-images.githubusercontent.com/49162938/66783931-9b6d1d00-eec8-11e9-8575-e8ae541b9f13.png)

It looks like that dbengine also needs some changes, but the apps.plugin also has problems.

**Comment by amoss** [comment]
I went back to check the memory mode on mechani.se - it defaulted to `save` on an install from the nightly. I've flipped it to `dbengine` and there is now some data in the `apps.cpu` chart for the `netdata` metric. It is still live on [mechanise](https://mechani.se/netdata/) if you want to take a look (with the dbengine).
![dbengine](https://user-images.githubusercontent.com/1043609/66817611-4834b200-ef3c-11e9-8471-07dcdd3f4938.png)
